packages/cozy-harvest-lib/src/components/AccountForm/ConnectionBackdrop.stories.jsx

```diff
@@ -0,0 +1,18 @@
+import { ConnectionBackdrop } from './ConnectionBackdrop'
+// import { UnrunnableAlert } from '../cards/UnrunnableAlert'
+
+const meta = {
+  component: ConnectionBackdrop,
+  arg: {
+    name: 'EDF'
+  }
+}
+
+export default meta
+
+export const Default = {
+  name: 'Default',
+  args: {
+    konnectorName: 'EDF'
```

Comment:
I don't think we need this argument because the parameter is `name` and it's already defined in the meta object